releasenotes/notes/fix_pwchebysev_constant_fx-93e8a3d2880f68ac.yaml

```diff
@@ -0,0 +1,8 @@
+---
+fixes:
+  - |
+    There was a problem when the function to be approximated was constant.
```

Comment:
Would you provide a bit of context in the fix note here. When its published as a release note, where this is what is being read, it talks about constant function bug but there is no other context here to understand it better.

When you update this there is also a minor spelling error ` funcions.`

Reply:
I've updated the release note and fixed the typo.